libraries/stdlib/jvm/src/kotlin/text/StringsJVM.kt

```diff
@@ -577,6 +602,16 @@ public actual fun String.decapitalize(): String {
     return if (isNotEmpty() && this[0].isUpperCase()) substring(0, 1).toLowerCase() + substring(1) else this
 }
 
+/**
+ * Returns a copy of this string having its first letter lowercased using [locale], or the original string,
+ * if it's empty or already starts with a lower case letter.
+ */
+@SinceKotlin("1.3")
+@ExperimentalStdlibApi
+public fun String.decapitalize(locale: Locale): String {
+    return if (isNotEmpty() && this[0].isUpperCase()) substring(0, 1).toLowerCase(locale) + substring(1) else this
```

Comment:
I've also changed `isUpperCase()` to `!isLowerCase()` here to cover cases, when the first char is in title case. See 8badfca4592dbac6faf0260159c73aed72285d69

Reply:
Ah, yep. Thanks!